client/components/sortable/sortable-list/__test__/sortableList.reducer.spec.js

```diff
@@ -0,0 +1,210 @@
+import { sortableReducer } from '../sortableList.reducer';
+import * as actions from '../sortableList.action';
+import items from '../items';
+
+describe('sortable list reducer', () => {
+	it('should return the initial state', () => {
+		const state = sortableReducer(undefined, {});
+		expect(state).toEqual({
+			loading: false,
+			items: [],
+			error: ''
+		});
+	});
+
+	it('should handle ITEMS_FETCH_REQUESTED', () => {
+		const state = {
+			loading: false,
+			items: [],
+			error: ''
+		};
+		const action = {
+			type: actions.ITEMS_FETCH_REQUESTED
+		};
+
+		let nextState = sortableReducer(state, action);
+		expect(nextState).toEqual({
+			loading: true,
+			items: [],
+			error: ''
+		});
+
+		nextState = sortableReducer({ ...state, error: 'some error!' }, action);
+		expect(nextState).toEqual({
```

Comment:
We try to do only one ``expect`` by ``it`` test case. Maybe we can change this to have nested ``describes`` and separate the expected behaviour for different scenarios in independent tests cases.